packages/mongo-nav/src/project-nav/ProjectNav.spec.tsx

```diff
@@ -28,7 +28,7 @@ const Products = {
   Charts: 'charts',
 };
 
-export type Products = typeof Products[keyof typeof Products];
```

Comment:
?

Reply:
Can't export from a spec file, but I dont believe we were importing this type anywhere